### ISSUE TITLE
Deprecated Warning: Using the last argument as keyword parameters is deprecated

### ISSUE THREAD

**Issue by mustangore** [railslove/cmxl#50]
Hey Bumi,

first of all: great job with cmxl :)

I was just about to play around with cmxl and received the following warning:
```shell
~/.rvm/gems/ruby-2.7.1@cmxl-test/gems/cmxl-1.4.6/lib/Cmxl.rb:39: warning: Using the last argument as keyword parameters is deprecated
```

I'm using ruby 2.7.1 and Cmxl 1.4.6.
For testing purpose I used your example described in `Simple usage` part of the README with fixtures file `mt940-iso8859-1.txt`.

Best wishes

**Comment by bumi** [comment]
@mustangore thanks for reporting. 
@JoeSouthan fixed this in #44  - a release is coming asap. I hope I will manage to do it tomorrow.

**Comment by mustangore** [comment]
Alright, thanks for the fast response.
